assets/js/search.js

```diff
@@ -251,17 +251,12 @@ document.addEventListener("DOMContentLoaded", function () {
     event.stopPropagation();
   });
 
-  overlayElementWeb.addEventListener("click", function (e) {
```

Comment:
There is no need to clearPopup, as there is clearInput() call from the HTML it self. see in displaySearchResults()